src/mscorlib/src/System/Runtime/InteropServices/NativeLibrary.Unix.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    // Contains UNIX-specific logic for NativeLibrary class.
+    public unsafe sealed partial class NativeLibrary
+    {
+        // The allowed mask values for the DllImportSearchPath.
+        // Non-Windows sytems only allow AssemblyDirectory and LegacyBehavior.
+        private const uint AllowedDllImportSearchPathsMask = (uint)DllImportSearchPath.AssemblyDirectory;
```

Comment:
Might be worth a comment that LegacyBehavior is 0, so it's implicitly in this list.  (So it doesn't look like a mismatch to the current comment)